draft-ietf-quic-qlog-main-schema.md

```diff
@@ -61,12 +62,14 @@ informative:
 
 --- abstract
 
-This document defines qlog, an extensible high-level schema for a standardized
-logging format. It allows easy sharing of data, benefitting common debug and
-analysis methods and tooling. The high-level schema is independent of protocol;
-separate documents extend qlog for protocol-specific data. The schema is also
-independent of serialization format, allowing logs to be represented in many
-ways such as JSON, CSV, or protobuf.
+qlog provides extensible structured logging for network protocols, allowing for
+easy sharing of data that benefits common debug and analysis methods and
+tooling. This document describes key concepts of qlog: formats, files, traces,
+events, and extension points. In this definition are the high-level log file
+schemas, and main event schema. Requirements and guidelines for creating
```

Comment:
Yeah it felt to me more lime this is stdlib than `main` but I refrained from changing it since the pr was some huge anyway. I'm open to a different value

Reply:
Discussed during editors' meeting. Should be `std` as namespace identifier.

Reply:
Discussed during editors' meeting.

Change to `generic` namespace and make category name `loglevel` instead :)